src/helpers/helpers.dom.ts

```diff
@@ -1,5 +1,7 @@
+import {ChartArea, Scale} from '../../types';
+import Chart from '../core/core.controller';
+import {ChartEvent} from '../types';
 import {INFINITY} from './helpers.math';
-
```

Comment:
(nit) IMO its good practice (and is used in this repository) to have an empty line between the imports and other code/comments. So I'd keep this line

Reply:
I agree with @kurkle , please restore empty line.

Reply:
✅